locales/en/plugin__kubevirt-plugin.json

```diff
@@ -754,6 +755,7 @@
   "Please don’t close this browser tab": "Please don’t close this browser tab",
   "Please provide hostname.": "Please provide hostname.",
   "Please wait, once the Data Volume has been created the data will start uploading into this Persistent Volume Claims.": "Please wait, once the Data Volume has been created the data will start uploading into this Persistent Volume Claims.",
+  "Please, make sure the configuration is correct before starting the VirtualMachine.": "Please, make sure the configuration is correct before starting the VirtualMachine.",
```

Comment:
We do not use "please" in our UIs or documentation. Also, the comma is incorrect. 

Suggested rewrite: "You must ensure that the configuration is correct before starting the VirtualMachine."